<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="P10102398" name="com.m3u8.test.TestBase" tests="9" failures="0" timestamp="25 Mar 2016 08:28:07 GMT" time="0.587" errors="1">
  <testcase name="testCompareTime" time="0.037" classname="com.m3u8.test.TestBase"/>
  <testcase name="testUnusualUrlResponse" time="0.033" classname="com.m3u8.test.TestBase"/>
  <testcase name="testisOnlyValue" time="0.024" classname="com.m3u8.test.TestBase"/>
  <testcase name="testResponse" time="0.051" classname="com.m3u8.test.TestBase"/>
  <testcase name="testSeg_noRepeat" time="0.025" classname="com.m3u8.test.TestBase"/>
  <testcase name="testCompareM3uPs" time="0.257" classname="com.m3u8.test.TestBase">
    <error type="java.io.IOException" message="m3u8和ps分片数不等,m3u8分片数为9ps分片数为12">
      <![CDATA[java.io.IOException: m3u8和ps分片数不等,m3u8分片数为9ps分片数为12
	at com.m3u8.basetool.GetTsSegNoRank.CompareM3uPs(GetTsSegNoRank.java:55)
	at com.m3u8.test.TestBase.testCompareM3uPs(TestBase.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
    </error>
  </testcase> <!-- testCompareM3uPs -->
  <testcase name="testFragmentMaxTime" time="0.125" classname="com.m3u8.test.TestBase"/>
  <testcase name="testExtinfDataType" time="0.002" classname="com.m3u8.test.TestBase"/>
  <testcase name="testSeg_noRank" time="0.033" classname="com.m3u8.test.TestBase"/>
</testsuite> <!-- com.m3u8.test.TestBase -->
